### DISCUSSION TITLE
'Microsoft.PowerShell-Preview-MSIX' and sub-package naming conventions

### DISCUSSION THREAD

**Discussion by bearmannl** [microsoft/winget-pkgs#18773] [Q&A] [answered]
@ItzLevvie, @OfficialEsco, currently there is a Microsoft.PowerShell-Preview-MSIX package. After fixing the PowerShell.Preview packages, I was wondering on (also for future reference) how to handle the PowerShell MSIX package.

First off, I know PowerShell has since been released via the Microsoft store, but I am also thinking of scenario's where one might want to install PowerShell on ARM based systems from command line at some point.

Now to the naming convention, I would suggest 'PowerShell.MSIX.Preview', making it clear that the MSIX package is an offshoot of the PowerShell package, with its own stable & preview features.

On the other hand, I would also be fine with removing the MSIX package distinction alltogether and just adding the MSIX packages as arm32/arm64 architecture packages to the relevant release version on the main PowerShell(.Preview) packages.

In the latter case I would still suggest tackling the above naming convention for other packages that do have proper 'sub packages' each with their own stable/beta/preview structures.

Would love to hear what you guys think!

**Comment by OfficialEsco** [comment, marked as answer]
I was confused about the Microsoft.PowerShell-Preview-MSIX package, i think they have stopped distributing the msix files and instead put it inside the zip(?)

The correct location would be in the same manifest as Microsoft.Powershell and Microsoft.Powershell.Preview, and maybe make a issue on https://github.com/PowerShell/PowerShell

Could probably add @denelon to this incase he got some extra information

**Reply by bearmannl** [reply]
Cool, that makes sense since it is a different installer type/architecture of the same package.

The PowerShell team discontinued the MSIX packages as of version [7.1.0-rc.2](https://github.com/PowerShell/PowerShell/issues/13866) by the way. Is [considered for version 7.3](https://github.com/PowerShell/PowerShell/milestone/103).
